### ISSUE TITLE
Debian 9

### ISSUE THREAD

**Issue by antton** [richardforth/apache2buddy#202]
Anyone is using the script on Debian 9?

I would like to know if it works or not.

Thanks

**Comment by richardforth** [comment]
Hi I've not tested it yet. There is a bug in the way that the php memory
limit is pulled out of the configuration which exists in all versions of
Debian and Ubuntu

**Comment by richardforth** [comment]
Reopened, accidentally quoted wrong issue when committing, my bad

**Comment by richardforth** [comment]
Hi,

I am testing this now, hoping to also iron out that php memory limit bug (#197) at the same time

**Comment by richardforth** [comment]
raising a new issue for debian 9 - unable to locate pid file:

```[ -- ] pidfile setting is /var/run/apache2$SUFFIX/apache2.pid.
[ !! ] Unable to locate pid file. Exiting.
```

**Comment by richardforth** [comment]
Raised a new issue:

Debian 9: issues with package updates #205

**Comment by richardforth** [comment]
Debian php.ini
#197 

Resolved in principal at least on debian 9, will commit and test in staging before merging with master

**Comment by richardforth** [comment]
All the bugs in debian 9 are now fixed, its still reporting the debian 9.0 is not supported:

[ -- ] We are attempting to discover the operating system type and version number ...
[ -- ] Distro: debian
[ -- ] Version: 9.0
[ -- ] Codename:
[ @@ ] New OS/Version verification checks are being worked on, you may get errors and teething problems. 02-04-2017
[ @@ ] Apologies for any inconvenience, this message will disappear when all issues resolved. 02-04-2017
[ OK ] This distro is supported by apache2buddy.pl.
[ !! ] This distro version (9.0) is not supported by apache2buddy.pl.


----

Added 9....

```
# https://wiki.debian.org/DebianReleases
        my @debian_supported_versions = ('7','8','9');
        my %dsv = map { $_ => 1 } @debian_supported_versions;
```
